src/audio/eq_fir/eq_fir.c

```diff
@@ -613,7 +614,9 @@ static int fir_cmd_set_data(struct comp_dev *dev,
 		/* Just copy the configuration. The EQ will be initialized in
 		 * prepare().
 		 */
-		memcpy(dst + offset, src, cdata->num_elems);
+		assert(!memcpy_s(dst + offset, cdata->num_elems +
+				 cdata->elems_remaining - offset, src,
+				 cdata->num_elems));
```

Comment:
maybe put the change to memcpy_s as a separate patch, this has nothing to do with check patch?

Reply:
@plbossart memcpy_s requirement has been recently added to checkpatch, but sure, I think the separate patch would be also OK.

Reply:
Done.